test/integration/dryrun/dryrun_test.go

```diff
@@ -289,8 +310,16 @@ func TestDryRun(t *testing.T) {
 
 			DryRunUpdateTest(t, rsc, name)
 			DryRunPatchTest(t, rsc, name)
-			DryRunScalePatchTest(t, rsc, name)
-			DryRunScaleUpdateTest(t, rsc, name)
+
+			runScaleTest := true
+			if verbs, ok := supportedVerbs[fmt.Sprintf("%s/scale", gvResource.Resource)]; ok {
+				_, runScaleTest = verbs["update"]
```

Comment:
IMO we should rather use the bool value or better, use a Set because we are not expecting false values